ui/src/components/Swap/Swap.jsx

```diff
@@ -466,10 +466,6 @@ const Swap = () => {
                   });
                   setSwapFrom(swapTo);
                   setSwapTo(swapFrom);
-                  setAssetExchange({
-                    ...assetExchange,
-                    marketRate: invertRatio(assetExchange.marketRate),
```

Comment:
I removed this because swapping the from/to already triggers the market rates to load again (and show a spinner) so there's no point, and it was causing a crash. I have a separate issue for showing the market rates better anyway.